functional-tests/hoverctl/config_test.go

```diff
@@ -17,13 +17,13 @@ var _ = Describe("When I use hoverctl", func() {
 		})
 
 		It("prints the contents of the config.yaml", func() {
-			functional_tests.Run(hoverctlBinary, "targets", "create")
+			functional_tests.Run(hoverctlBinary, "targets", "new", "config-test")
```

Comment:
Why change to `new`? Having a keyword that reflects an action like these is good: 
``` 
hoverctl targets create target-name
hoverctl targets update target-name
hoverctl targets delete target-name
 ```